common/priceable.go

```diff
@@ -9,3 +9,8 @@ this resource.
 type Priceable interface {
 	GetHourlyPrice(ctx context.Context) float64
 }
+
+type TerraformPriceableAsset interface {
```

Comment:
Is there a world where this gets merges into the `Priceable` interface? I could see the first method `Keys()` becoming: `GetSupportedTerraformResourceList` and the well I guess generating a Priceable from a priceable is not a great idea. But if it was, `GetPriceables` would be the next method suggestion. Also I forgot what I said on the phone, but I thought I suggested a different name for the interface as well.